iree/vm/BUILD

```diff
@@ -185,7 +185,6 @@ cc_library(
         "bytecode_module.h",
     ],
     deps = [
-        ":bytecode_op_table_gen",
```

Comment:
This move shouldn't be java-bindings specific. Move the autogenerated file here to `iree/vm/` and add as a header to this rule where the dep is now removed.

Reply:
I see now, thank you!  Fixed